superset/Dockerfile

```diff
@@ -5,5 +5,15 @@ USER root
 RUN pip install psycopg2
 
 RUN pip install psycopg2-binary
+
+RUN superset superset fab create-admin \
+               --username admin \
+               --firstname Superset \
+               --lastname Admin \
+               --email admin@superset.com \
+               --password admin
```

Comment:
We should probably force users to specify an admin password in their environment variables.